fonts.gradle

```diff
@@ -25,19 +25,6 @@ afterEvaluate {
     }
 
     android.applicationVariants.all { def variant ->
-        def targetName = variant.name.capitalize()
-        def lintVitalAnalyzeTask = tasks.findByName("lintVitalAnalyze${targetName}")
-      
-        if (lintVitalAnalyzeTask) {
-        lintVitalAnalyzeTask.dependsOn(fontCopyTask)
-        }
-
-        def generateReportTask = tasks.findByName("generate${targetName}LintVitalReportModel")
-        if (generateReportTask) {
-            generateReportTask.dependsOn(fontCopyTask)
-        }
-      
-        def generateAssetsTask = tasks.findByName("generate${targetName}Assets")
-        generateAssetsTask.dependsOn(fontCopyTask)
-      }
+        variant.preBuildProvider.configure { it.dependsOn(fontCopyTask) }
```

Comment:
What's the difference between this and having the `preBuild` task depend on it? When you tried this out, did you clear gradle cache in-between, because when I tried it didn't work properly.